slick.grid.js

```diff
@@ -4768,19 +4768,15 @@ if (typeof Slick === "undefined") {
         activeRow = cell.row;
         activeCell = activePosX = activeCell = activePosX = getCellFromNode(activeCellNode);
 
-        $activeCellNode.addClass("active");
-        if (rowsCache[activeRow]) {
-          $(rowsCache[activeRow].rowNode).addClass('active');
-        }
-
-
         if (opt_editMode == null) {
           opt_editMode = (activeRow == getDataLength()) || options.autoEdit;
         }
 
         if (options.showCellSelection) {
-        $(activeCellNode).addClass("active");
-        $(rowsCache[activeRow].rowNode).addClass("active");
+          $activeCellNode.addClass("active");
+          if (rowsCache[activeRow]) {
```

Comment:
Without this conditional it's possible to receive `Uncaught TypeError: Cannot read property 'rowNode' of undefined`

Reply:
Sure thing, good idea 👍

Reply:
note: so this is a breaking change, but I'd view it as one that reverses an earlier breaking change as a result of an introduced bug. Behaviour should now be back to as intended.